packages/components/src/components/carousel/carousel.stories.ts

```diff
@@ -51,199 +62,178 @@ export const Default = {
 
 /**
  * Use the `variant` attribute to select the pagination format.
+ *
+ * - `Number` variant (default): can be used for all use cases.
+ * - `Dot` variant: can only be used for up to 5 items total.
  */
 
 export const Variant = {
-  parameters: { controls: { exclude: 'variant' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'variant', values: ['dot', 'number'] }
-      },
-      args
-    });
-  }
+  name: 'Variant',
+  render: () => html`
+    <div class="flex gap-12">
+      <div>
+        <sd-carousel variant="dot">
+          <sd-carousel-item>
+            <div class="slot slot--border slot--text h-16">Default slot 1</div>
+          </sd-carousel-item>
+          <sd-carousel-item>
+            <div class="slot slot--border slot--text h-16">Default slot 2</div>
+          </sd-carousel-item>
+          <sd-carousel-item>
+            <div class="slot slot--border slot--text h-16">Default slot 3</div>
+          </sd-carousel-item>
+        <sd-carousel>
```

Comment:
Please add closing tags to all instances of sd-carousel. 
This seems to be the cause of the bug on #1269.

Reply:
Thanks @Vahid1919, nice catch